README.md

```diff
@@ -361,7 +362,6 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [crystal-snmp](https://github.com/spider-gazelle/crystal-snmp) - An SNMP implementation with version 1, 2c and 3 support
  * [fast_irc.cr](https://github.com/RX14/fast_irc.cr) - Fast IRC parser/generator
  * [jwt](https://github.com/crystal-community/jwt) - Implementation of JWT (JSON Web Token)
- * [mDNS](https://github.com/spider-gazelle/mdns) - DNS Service Discovery and multicast DNS
```

Comment:
Prob want to rebase on master assuming this isn't intended.

Reply:
Good catch, thanks!